config/dev.exs

```diff
@@ -3,9 +3,9 @@ use Mix.Config
 # Configure your database
 config :ag, AG.Repo,
   username: "postgres",
-  password: "postgres",
+  password: "",
```

Comment:
@teerawat1992 please update these to read from env file hai gu noi

Reply:
How about simply set env variables in docker-compose instead

```
environment:
      POSTGRES_USER: postgres
      POSTGRES_PASSWORD: postgres
      POSTGRES_DB: ag_dev
```

Reply:
How about simply set env variables in docker-compose instead

```
environment:
      POSTGRES_USER: postgres
      POSTGRES_PASSWORD: postgres
      POSTGRES_DB: ag_dev
```

Reply:
it is fine but here should have changed to read from env vars, btw.

Reply:
just got it, deaw add hai.